<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgetPass</class>
 <widget class="QDialog" name="ForgetPass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="forgetNLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="forgetELabel">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>90</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>邮箱：</string>
   </property>
  </widget>
  <widget class="QLabel" name="forgetLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>6</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; font-style:italic;&quot;&gt;找回密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="forgetNLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="forgetELineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="forgetExitButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>35</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forgetOkButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>找 回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forgetNoButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>放 弃</string>
   </property>
  </widget>
  <widget class="QLabel" name="ForgetWarnLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>158</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="forgetTipLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>邮箱格式为qq邮箱格式</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>forgetNLineEdit</tabstop>
  <tabstop>forgetELineEdit</tabstop>
  <tabstop>forgetOkButton</tabstop>
  <tabstop>forgetNoButton</tabstop>
  <tabstop>forgetExitButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
